src/sentry/grouping/strategies/newstyle.py

```diff
@@ -223,6 +218,16 @@ def get_function_component(function, platform, legacy_function_logic,
                     hint='isolated function'
                 )
 
+    elif javascript_fuzzing and behavior_family == 'javascript':
+        # This changes Object.foo or Foo.foo into foo so that we can
+        # resolve some common cross browser differences
+        new_function = func.rsplit('.', 1)[-1]
```

Comment:
Most browsers do not send `Foo.foo`, so we are always limited to just `foo` on a lot of these. I rather see all things grouped together than to theoretically have better groups on one. Additionally you normally do not see the class name, just `Object.foo`.

Reply:
How would this effect applications using frameworks like ember, where `set()` and `get()` are very common and on basically every object?